lab_2_retrieval_w_bm25/start.py

```diff
@@ -29,44 +29,34 @@ def main() -> None:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
+    result = None
     clean_documents = []
     for document in documents:
         tokenized_document = tokenize(document)
-        if not tokenized_document:
-            return None
-        clean_document = remove_stopwords(tokenized_document, stopwords)
-        if not clean_document:
-            return None
-        clean_documents.append(clean_document)
+        if tokenized_document:
+            clean_document = remove_stopwords(tokenized_document, stopwords)
+            if clean_document:
+                clean_documents.append(clean_document)
     vocab = build_vocabulary(clean_documents)
     if not vocab:
-        return None
-    idf = calculate_idf(vocab, clean_documents)
-    if not idf:
-        return None
+        idf = calculate_idf(vocab, clean_documents)
```

Comment:
why it is after ```if not vocab:```  ?

Reply:
actually i dont know why its there...
fixed